src/uct/api/v2/uct_v2.h

```diff
@@ -1050,6 +1050,83 @@ int uct_iface_is_reachable_v2(uct_iface_h iface,
                               const uct_iface_is_reachable_params_t *params);
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Parameters passed to @ref uct_md_query_tl_resources_v2.
+ */
+typedef struct uct_md_query_tl_resources_params {
+    /**
+     * Mask of valid fields which currently has to be zero.
```

Comment:
```suggestion
     * Mask of valid fields which currently have to be zero.
```

Reply:
I am not a native english speaker, but I think "have" is for I/We/You/They. Here it is "It" (the Mask) that should be zero. Replaced by "must" anyways ;)

Reply:
Yeah, I am not a native english speaker too :)

I thought that in that case the subject is "fields" which are plural and not "mask". Anyway, replacing by "must" works in both cases. Thanks!